### ISSUE TITLE
File Groups With Files Nested 2 folders below make the folder appear at top level

### ISSUE THREAD

**Issue by aure** [yonaskolb/XcodeGen#925]
While I would love it if the blobbing solution from this issue: https://github.com/yonaskolb/XcodeGen/issues/676 were implemented, in the meantime I can list all the files I need to include as follows:

```
fileGroups:
  - "README.md"
  - "Common/README.md"
  - "Common/MIDI/README.md"
```
However, when I do that, the README.md file for the first two lines appears correctly, but lower than, that or when the parent directory for the file is not already in a path, the folder containing the README gets bumped up too high in the Xcode file brower as show here:

![image](https://user-images.githubusercontent.com/13122/89670459-15c2bf80-d896-11ea-9a2c-3449b739a337.png)

In other words MIDI is no longer a subdirectory of Common in the generated Xcode project as soon as I add the `Common/MIDI/README.md` fileGroup.
